### ISSUE TITLE
Missing `proto-google-cloud-bigquerydatatransfer-v1` when importing `google-cloud-bigquerydatatransfer` ?

### ISSUE THREAD

**Issue by norbjd** [googleapis/java-bigquerydatatransfer#230]
Hello :wave:,

Instructions in the [README](https://github.com/googleapis/java-bigquerydatatransfer/blob/76505aa9bb703e6426fd7c3dd23118b3aac7dba7/README.md#quickstart) to show how to use this library seems to be incomplete. To use the dependency `com.google.cloud:google-cloud-bigquerydatatransfer`, `com.google.cloud:proto-google-cloud-bigquerydatatransfer-v1` should also be included :

```
<dependency>
      <groupId>com.google.api.grpc</groupId>
      <artifactId>proto-google-cloud-bigquerydatatransfer-v1</artifactId>
</dependency>
```

Otherwise, there are many missing classes (like `TransferRun`, `TransferConfig`, ...), and `com.google.cloud:google-cloud-bigquerydatatransfer` could not be used on its own.

By the way, I checked in the `pom.xml`, I don't understand why the `com.google.cloud:proto-google-cloud-bigquerydatatransfer-v1` is not included when using `com.google.cloud:google-cloud-bigquerydatatransfer`. For information, if that can help, following the README, `mvn dependency:tree -Dverbose` gives me :

```
[INFO] com.example:my-project:jar:1.0-SNAPSHOT
[INFO] +- com.google.cloud:google-cloud-bigquerydatatransfer:jar:1.0.0:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```

Is this deliberate or is this a bug ?

Thanks.

**Comment by stephaniewang526** [comment]
There is no need to include proto module separately. It is in google-cloud-bigquerydatatransfer [pom.xml](https://github.com/googleapis/java-bigquerydatatransfer/blob/19feb03b71e77d943cb8d05beb44739e8d16dca3/google-cloud-bigquerydatatransfer/pom.xml#L51).

**Comment by norbjd** [comment]
@stephaniewang526 I should have mentioned that the issue appears **only** when using the bom `com.google.cloud:libraries-bom:8.0.0`, as described in the README : https://github.com/googleapis/java-bigquerydatatransfer/tree/v1.0.0#quickstart. You can reproduce the issue with this sample `pom.xml` :

```
<project>
  <modelVersion>4.0.0</modelVersion>

  <groupId>com.mycompany.app</groupId>
  <artifactId>my-app</artifactId>
  <version>1</version>

  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>com.google.cloud</groupId>
        <artifactId>libraries-bom</artifactId>
        <version>8.0.0</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>

  <dependencies>
    <dependency>
      <groupId>com.google.cloud</groupId>
      <artifactId>google-cloud-bigquerydatatransfer</artifactId>
    </dependency>
  </dependencies>
</project>
```

`mvn dependency:tree -Dverbose` shows :

```
[INFO] Scanning for projects...
[INFO] 
[INFO] ----------------------< com.mycompany.app:my-app >----------------------
[INFO] Building my-app 1
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for com.google.cloud:google-cloud-bigquerydatatransfer:jar:1.0.0 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
[INFO] 
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ my-app ---
[WARNING] Invalid POM for com.google.cloud:google-cloud-bigquerydatatransfer:jar:1.0.0, transitive dependencies (if any) will not be available, enable debug logging for more details
[INFO] com.mycompany.app:my-app:jar:1
[INFO] \- com.google.cloud:google-cloud-bigquerydatatransfer:jar:1.0.0:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.781 s
[INFO] Finished at: 2020-07-07T07:58:31+02:00
[INFO] ------------------------------------------------------------------------
```

I believe that the warning is important here :

```
[WARNING] The POM for com.google.cloud:google-cloud-bigquerydatatransfer:jar:1.0.0 is invalid, transitive dependencies (if any) will not be available
```

Running `mvn` in debug mode (`mvn -X dependency:tree -Dverbose`) shows :

```
[WARNING] Invalid POM for com.google.cloud:google-cloud-bigquerydatatransfer:jar:1.0.0, transitive dependencies (if any) will not be available, enable debug logging for more details: Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for javax.annotation:javax.annotation-api:jar is missing. @ line 827, column 2
```

**Comment by stephaniewang526** [comment]
Looks like the version of bigquerydatatransfer is outdated in libraries-bom. It should be v1.0.1 but it is v1.0.0. We fixed this known issue in v1.0.1.

Please use the artifact directly while we address this issue in libraries-bom. Thanks!

**Comment by norbjd** [comment]
@stephaniewang526 Ok, thanks ! Should I open an issue somewhere for `libraries-bom` ?

**Comment by stephaniewang526** [comment]
I chatted with them yesterday -- since the new version of bigquerydatatransfer did not get published in Maven Central correctly, it is not really a problem with the libraries-bom. We just released v1.0.2 yesterday and it will go into the libraries-bom soon so this issue will be fixed then.
